Joins-Lab/app/src/main/java/com/ezequiel/student/joins_lab/SQLHelper.java

```diff
@@ -0,0 +1,164 @@
+        ContentValues values = new ContentValues();
+        values.put(COL_SSN, employee.getSSN());
+        values.put(COL_FIRST_NAME, employee.getFirstName());
+        values.put(COL_LAST_NAME, employee.getLastName());
+        values.put(COL_YEAR_OF_BIRTH, employee.getYearOfBirth());
+        values.put(COL_CITY, employee.getCity());
+        db.insertOrThrow(EMPLOYEE_TABLE_NAME, null, values);
+    }
+
+    public void insertRowJob(Job job){
+        SQLiteDatabase db = getWritableDatabase();
+        ContentValues values = new ContentValues();
+        values.put(COL_SSN, job.getEmpSSN());
+        values.put(COL_COMPANY, job.getCompany());
+        values.put(COL_SALARY, job.getSalary());
+        values.put(COL_EXPERIENCE, job.getExperience());
+        db.insertOrThrow(JOB_TABLE_NAME, null, values);
+    }
+
+    public List<String> getEmpFromMacys(){
+        SQLiteDatabase db = getReadableDatabase();
+
+        String query = "SELECT " + COL_FIRST_NAME + ", " + COL_LAST_NAME +
+                " FROM " + EMPLOYEE_TABLE_NAME + " JOIN " + JOB_TABLE_NAME +
+                " ON " + EMPLOYEE_TABLE_NAME+"."+COL_SSN + " = " + JOB_TABLE_NAME+"."+COL_SSN +
+                " WHERE " + COL_COMPANY + " = 'Macys'";
+
+        Cursor cursor = db.rawQuery(query, null);
+
+        List<String> names = new ArrayList<>();
+
+        if (cursor.moveToFirst()){
+            while (!cursor.isAfterLast()){
+                names.add(cursor.getString(cursor.getColumnIndex(COL_FIRST_NAME))+" "+
+                        cursor.getString(cursor.getColumnIndex(COL_LAST_NAME)));
+                cursor.moveToNext();
+            }
+        }
+        cursor.close();
+        return names;
+    }
+
+    public List<String> getEmpFromBoston(){
+        SQLiteDatabase db = getReadableDatabase();
+
+        String query = "SELECT " + COL_FIRST_NAME + ", " + COL_LAST_NAME +
+                " FROM " + EMPLOYEE_TABLE_NAME + " JOIN " + JOB_TABLE_NAME +
+                " ON " + EMPLOYEE_TABLE_NAME+"."+COL_SSN + " = " + JOB_TABLE_NAME+"."+COL_SSN +
+                " WHERE " + COL_CITY + " = 'Boston'";
+
+        Cursor cursor = db.rawQuery(query, null);
+
+        List<String> names = new ArrayList<>();
+
+        if (cursor.moveToFirst()) {
+            while (!cursor.isAfterLast()) {
+                names.add(cursor.getString(cursor.getColumnIndex(COL_FIRST_NAME)) + " " +
+                        cursor.getString(cursor.getColumnIndex(COL_LAST_NAME)));
+                cursor.moveToNext();
+            }
+        }
+        cursor.close();
+        return names;
+    }
+
+    public List<String> getHighSalary(){
+        SQLiteDatabase db = getReadableDatabase();
+
+        String query = "SELECT " + COL_FIRST_NAME + ", " + COL_LAST_NAME +
+                " FROM " + EMPLOYEE_TABLE_NAME + " JOIN " + JOB_TABLE_NAME +
+                " ON " + EMPLOYEE_TABLE_NAME+"."+COL_SSN + " = " + JOB_TABLE_NAME+"."+COL_SSN +
+                " ORDER BY " + COL_SALARY + " ";
```

Comment:
You'll want to add `DESC` at the end of the `ORDER BY` clause. By default it sorts in _ascending_ order from smallest to largest, but in this case you want _descending_ order from largest to smallest.